media-source/MediaSource-addSourceBuffer_readyState.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>readyState attribute not in the "open" state</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+var video = document.createElement('video');
+var mimes = ['video/webm; codecs="vorbis,vp8"', 'video/mp4'];
```

Comment:
ditto: use one of the *_TYPE  provided by mediasource-util.js (or just use mediasource_test...